lib/srv/db/redis/test.go

```diff
@@ -50,6 +52,13 @@ func SkipPing(skip bool) ClientOptions {
 	}
 }
 
+// WithTimeout overrides test client's default timeout.
+func WithTimeout(timeout time.Duration) ClientOptions {
```

Comment:
This doesn't appear to be used, do we need it?

Reply:
It's very useful when debugging flakiness like this. Will keep it just in case.